api/src/main.rs

```diff
@@ -90,7 +90,7 @@ async fn openapi(request_info: RequestInfo) -> Option<String> {
 struct Index(String);
 
 #[rocket::get("/")]
-async fn index(user: User, issuers: &State<Issuers>) -> HttpResult<Index> {
+async fn index(user: User, issuers: &State<Issuers>, config_state: &State<Config>) -> HttpResult<Index> {
```

Comment:
I usually don't suffix the state aspect.

```suggestion
async fn index(user: User, issuers: &State<Issuers>, config: &State<Config>) -> HttpResult<Index> {
```